src/Sylius/Bundle/ApiBundle/Validator/Constraints/ShopUserExists.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/** @experimental */
+final class ShopUserExists extends Constraint
+{
+    public string $message = 'sylius.shop_user.not_found';
+
+    public function validatedBy(): string
+    {
+        return 'sylius_api_shop_user_exists';
+    }
+}
```

Comment:
Should I delete this validator even if it is no longer used?

It can be used if someone wanted it, and he/she will have to simply add it to an XML validation file for example.

Reply:
Why is this in the ApiBundle anyway? I don't see anything API specific in this validator.

Reply:
Indeed, none of the new validators are API-specific.

Reply:
Agree too...